doc-content/enterprise-only/admin-and-config/proc-configuring-kie-server-persistence.adoc

```diff
@@ -0,0 +1,64 @@
+[id='proc-configuring-kie-server-persistence_{context}']
+
+= Configuring {KIE_SERVER} Persistence
+
+You can configure the {KIE_SERVER} persistence by passing hibernate or JPA parameters as system properties.
+
+To pass the hibernate or JPA parameters, use the following prefixes in your system properties:
```

Comment:
I would rephrase this a bit, these prefixes are not prepended by a user, rather it means that properties with these prefixes are recognized by a KIE Server. So users can use every JPA or Hibernate parameters with these prefixes. Also see the original JIRA which says so: `kie-server will take properties with following prefix into consideration:`.

Reply:
Updated